corehq/apps/domain/forms.py

```diff
@@ -498,7 +504,7 @@ def save(self, request, domain):
         domain.hr_name = self.cleaned_data['hr_name']
         domain.project_description = self.cleaned_data['project_description']
         domain.default_mobile_ucr_sync_interval = self.cleaned_data.get('mobile_ucr_sync_interval', None)
-        domain.default_geocoder_location = json.loads(self.cleaned_data.get('default_geocoder_location','{}'))
+        domain.default_geocoder_location = self.clean_default_geocoder_location()
```

Comment:
`clean_default_geocoder_location` gets called automatically and will populate `cleaned_data` with the clean value, the dict. So this can just be `self.cleaned_data['default_geocoder_location']`

Reply:
Ah, was wondering why are other clean methods not being called anywhere. Thanks Jenny. Made the change.